api/client.go

```
@@ -243,11 +247,10 @@ func (c Client) HasMinimumScopes(hostname string) (bool, error) {
 	}
 
 	if len(errorMsgs) > 0 {
-		return false, errors.New(strings.Join(errorMsgs, ";"))
+		return false, &MissingScopesError{error: errors.New(strings.Join(errorMsgs, ";"))}
```

Nit: something that has _just_ occurred to me, although it hasn't been introduced in this PR: `HasMinimumScopes` was originally designed to return a bool confirming whether or not the token checks out, but we also consistently return an error object describing which scopes are missing. It seems that the bool return value is redundant and that we can just return a single error value from the function?

---

good catch! cleaned up